### DISCUSSION TITLE
Add Raptoreums GhostRider Algorithm

### DISCUSSION THREAD

**Discussion by Raptor3um** [coinfoundry/miningcore#852] [Ideas]
Raptoreum launched mainnet 4 days ago and the GhostRider Algorithm is now public:
https://github.com/npq7721/gr_hash

Been a fan of Miningcore for a longtime would love to see GR on it.

**Comment by Konstantin35** [comment]
at the moment oliverw has scored on his project you can find the continuation here https://discord.gg/7dyvNx4H7S

**Comment by Raptor3um** [comment]
Got it, thank you.
